### DISCUSSION TITLE
Windows non AD Desktop Names

### DISCUSSION THREAD

**Discussion by lfb-sanger** [gravitational/teleport#34540] [Q&A]
Hi,

I've configured several Windows servers using the documentation and all are configured and working as expected, the only question I have is that the names of the servers are showing up with random ID before their hostname and I wondered if there was a way to just display their hostname and remove the random ID?

Thank you,

Luke

**Comment by zmb3** [comment]
Not yet, but we will be adding this functionality soon. Feel free to follow https://github.com/gravitational/teleport/issues/30886 for updates.
